core/templates/services/question-validation.service.ts

```diff
@@ -53,6 +53,24 @@ export class QuestionValidationService {
     );
   }
 
+  areAnswerGroupsNotEqual(question: Question): boolean {
```

Comment:
Two things:

1. Probably you want to frame this as areAnswerGroupsEqual (otherwise you have a double negation and it's a bit more confusing).
2. Should this be general functionality for all interactions? If so can we put it in a common place? It does not seem specific to this interaction.